app/src/main/res/layout/dialog_editprofile.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="20dp"
+    android:background="@drawable/button_round_corner"
+    >
+
+    <EditText
+        android:id="@+id/editEmail"
```

Comment:
I believe the naming convertion for xml files is [snake_case](https://en.wikipedia.org/wiki/Snake_case)

e.g. edit_email instead of editEmail

try keep these convention in mind so you won't be naming variables using different conventions within the same file